.github/workflows/tests.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           - 7.1.1
           - 7.0.3
 
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
Why is this being changed?

Reply:
20.04 is being deprecated and they're running brown outs which results in test failures.